mne/label.py

```diff
@@ -2666,3 +2669,43 @@ def select_sources(subject, label, location='center', extent=0.,
                           hemi=new_label.hemi, name=name, subject=subject)
 
     return new_label
+
+
+def find_label_in_annot(pos, subject='fsaverage', annot="aparc.a2005s+aseg",
+                         subjects_dir=None):
+    """
+    Find name in atlas for given MRI coordinates.
+
+    Parameters
+    ----------
+    pos : vector of x,y,z coordinates in MRI space
+    subject : MRI subject name
+    annot : parcellation file name
+    subjects_dir : path to MRI subjects directory
+
+    Returns
+    -------
+    label : string - parcellation label
+
+
+    .. versionadded:: 0.24
+    """
+    import nibabel
+    if subjects_dir is None:
+        subjects_dir = get_subjects_dir(None)
+    atlas_fname = os.path.join(subjects_dir, subject, 'mri', annot + '.mgz')
+    parcellation_img = nibabel.load(atlas_fname)
+
+    # Load freesurface atlas LUT
+    lut_inv_dict = read_freesurfer_lut()[0]
+    label_lut = {v: k for k, v in lut_inv_dict.items()}
+
+    # Find voxel for dipole position
+    mri_vox_t = np.linalg.inv(parcellation_img.header.get_vox2ras_tkr())
+    vox_dip_pos_f = mne.transforms.apply_trans(mri_vox_t, pos)
+    vox_dip_pos = vox_dip_pos_f.astype(int)
```

Comment:
might want `np.rint()` here?  `.astype(int)` will turn a coordinate of `5.99` into `5`